app/src/main/java/com/keylesspalace/tusky/components/report/adapter/StatusesDataSource.kt

```diff
@@ -72,14 +72,18 @@ class StatusesDataSource(private val accountId: String,
         retryBefore = null
         retryInitial = null
         initialLoad.postValue(NetworkState.LOADING)
-        mastodonApi.statusObservable(params.requestedInitialKey).zipWith(
-                mastodonApi.accountStatusesObservable(accountId, params.requestedInitialKey, null, params.requestedLoadSize - 1, true),
-                BiFunction { status: Status, list: List<Status> ->
-                    val ret = ArrayList<Status>()
-                    ret.add(status)
-                    ret.addAll(list)
-                    return@BiFunction ret
-                })
+        if (params.requestedInitialKey == null) {
```

Comment:
What is this key and why this check is needed now?

Reply:
Its the id of the status that is being reported. Null means user did not select report on a toot but on a profile.